src/htmx.js

```diff
@@ -1232,6 +1234,18 @@ return (function () {
             return result;
         }
 
+        function consumeCSSSelector(tokens) {
+            var result = "";
+            if (tokens.length > 0 && tokens[0].match(COMBINED_SELECTOR_START)) {
+                tokens.shift();
+                result += consumeUntil(tokens, COMBINED_SELECTOR_END).trim();
```

Comment:
Did this because it felt more logical that the variable `result` always exists, as we're going to return it, figured you'd rather return an empty string if the code is wrong than error out.
Either way, I adjusted it now to be as you suggest.